README.md

```diff
@@ -57,8 +57,8 @@ if (isCI) {
 
 | Variable   | Description                                                                                            |
 | ---------- | ------------------------------------------------------------------------------------------------------ |
-| `name`     | CI service Commercial name (e.g. `Travis CI`, `CircleCI`, `GitLab CI/CD`)                              |
-| `service`  | Standardized CI service name (e.g. `travis`, `circleci`, `gitlab`)                                     |
+| `name`     | CI service Commercial name (e.g. `Travis CI`, `CircleCI`, `GitLab CI/CD`, `Buddy`)                              |
```

Comment:
this list is intended to be an example, not complete. i dont think buddy needs to be added here